index.html

```diff
@@ -111,7 +111,7 @@ <h2>Books</h2>
 
 			<div id="projects">
 
-				<!-- January 2020 -->
```

Comment:
This comment is actually for me to remember when these projects were featured. Better not to change this.